src/cdt.rs

```diff
@@ -1995,6 +2012,26 @@ mod test {
         Cdt::bulk_load_cdt_stable(vertices, vec![[3, 2], [5, 4], [7, 6]])
     }
 
+    #[test]
+    fn get_try_cdt_for_duplicate_vertices() -> Result<(), InsertionError> {
+        let vertices = vec![
+            Point2::new(0.0, -10.0),
+            Point2::new(76.0, 0.0),
+            Point2::new(76.0, 0.0), // Duplicated vertex
+            Point2::new(20.0, -30.0),
+            Point2::new(45.0, 25.0),
+            Point2::new(32.0, -35.0),
+            Point2::new(60.0, 20.0),
+            Point2::new(60.0, -30.0),
+            Point2::new(50.0, -34.0),
+        ];
+        let (_, conflicting_edges) =
+            Cdt::try_bulk_load_cdt_stable(vertices, vec![[3, 2], [5, 4], [7, 6]])?;
+        // Hardcoded values, may change if CDT algorithm change
+        assert_eq!(&conflicting_edges, &[[6, 7,], [4, 5,]]);
```

Comment:
Can you add an assert that checks that the non-conflicting edge was added successfully? E.g. `assert_eq!(cdt.num_constraints, 1)` (I think?)